amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -322,6 +322,10 @@ type Colony @model {
   version: Int!
   actions: [ColonyAction] @hasMany
   metadata: ColonyMetadata @hasOne(fields: ["id"])
+  # @TODO This should not be fetched upfront
+  # It should be retrieved on demand at the earliest occasion it's needed
+  # ie: when opening a UAC modal
+  roles: [ColonyRole] @hasMany
```

Comment:
Would this comment be more appropriate in the CDapp where we actually fetch it?

Reply:
I actually don't know why I left a todo here? But the comment should be also here, although I agree I need to move it to the Dapp as well